src/i18n/translations.ts

```diff
@@ -235,6 +239,10 @@ export default {
         infoMenuTitle: 'About',
         logoutMenuTitle: 'Logout',
         loginMenuTitle: 'Login'
+      },
+      WmsTimeSlider: {
+        title: 'Time of recording',
```

Comment:
Probably a more generic label like `Time reference`? Recording refers to an earth observation dataset, but it might be any time related dataset like a weather forecast or similar.